docs/Coordinator.md

```diff
@@ -27,6 +30,39 @@ Module that provides access to information about Contxt
 | sdk | <code>Object</code> | An instance of the SDK so the module can communicate with other modules |
 | request | <code>Object</code> | An instance of the request module tied to this module's audience. |
 
+<a name="Coordinator+activateNewUser"></a>
+
+### contxtSdk.coordinator.activateNewUser(userId, user) ⇒ <code>Promise</code>
+Activates a new user
+
+API Endpoint: '/users/:userId/users'
```

Comment:
Does this need to be `/users/:userId/activate`?

Reply:
Good catch, thanks 👍